synapse/tests/test_lib_cell.py

```diff
@@ -1370,7 +1370,7 @@ async def _iterNewDup(self, user, name=None, remove=False):
 
                     fail = s_t_utils.alist(proxy.iterNewBackupArchive('alreadyrunning', remove=True))
                     await self.asyncraises(s_exc.BackupAlreadyRunning, fail)
-                    await asyncio.wait_for(task, 5)
+                    task.cancel()
```

Comment:
?

Reply:
removed, doesn't need to be changed here